synthetics/test_schema.go

```diff
@@ -4,46 +4,53 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+// TODO(dfurman): mark required attributes, when they are specified in the OpenAPI definitions
+// TODO(dfurman): provide descriptions, when they are specified in the OpenAPI definitions
+
 func makeTestSchema(mode schemaMode) map[string]*schema.Schema {
 	return map[string]*schema.Schema{
 		"id": {
```

Comment:
Can/should this be `idKey` to keep things consistent?

Reply:
I'll change that